lib/controllers/error_controller.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:unifit/models/log_controller.dart';
+
+class ErrorHandlerController {
+  static String signUpErrorHandling(FirebaseAuthException error) {
+    late String errorMessage;
+    late String errorLog;
+
+    switch (error.code) {
+      case "email-already-in-use":
```

Comment:
Please add all application strings to utils.constants.AppStrings, this turns way easier the string reutilization and refactoring.